runners/runner-local/src/server.rs

```diff
@@ -0,0 +1,158 @@
+use tracing::warn;
+
+use crate::error::LocalRunnerError;
+use crate::manager::IndexerManager;
+
+use apibara_runner_common::runner::v1::{
+    indexer_runner_server, CreateIndexerRequest, DeleteIndexerRequest, GetIndexerRequest, Indexer,
+    ListIndexersRequest, ListIndexersResponse, UpdateIndexerRequest,
+};
+
+use error_stack::{Report, ResultExt};
+use tokio::process::Child;
+use tonic::{Request, Response};
+
+// TODO: Ctrl-C doesn't work if the server has a lock on the indexers mutex
```

Comment:
You can (probably) work around this by using a tokio's mutex and passing the cancellation token around.